src/main/java/gameboy/gamepack/racinggame/view/InputView.java

```diff
@@ -1,25 +1,74 @@
 package gameboy.gamepack.racinggame.view;
 
-import java.util.InputMismatchException;
-import java.util.Scanner;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import gameboy.gamepack.racinggame.data.vo.Name;
+import gameboy.gamepack.racinggame.exception.InvalidRacerNameException;
 
 public class InputView {
 
-    public static int inputNumber(String ask) {
+    private static final String CARS_NAME_SEPARATOR = ",";
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public static int inputRaceCount() {
+        return inputNumber("시도할 회수는 몇 회 인가요?");
+    }
+
+    public static List<Name> inputCarsName() {
+        return inputCarsName("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+    }
+
+    private static List<Name> inputCarsName(String ask) {
+        try {
+            List<String> carNames = split(inputText(ask), CARS_NAME_SEPARATOR);
+            validDuplicateName(carNames);
+            return carNames.stream()
+                .map(Name::new)
+                .toList();
+        } catch (InvalidRacerNameException e) {
+            return inputCarsName(e.getMessage());
+        }
+    }
+
+    private static List<String> split(String text, String separator) {
+        List<String> carNames = List.of(text.split(separator))
+            .stream()
+            .map(String::trim)
+            .collect(Collectors.toList());
+        return carNames;
```

Comment:
```suggestion
        return Stream.of(text.split(separator))
                .map(String::trim)
                .toList();
```
같은 코드입니다.

split된 배열을 리스트로 만들고 stream으로 변경하는 방식보다는
Stream으로 바로 넣어주는 게 더 좋을 것 같아요
그리고 `.collect(Collectors.toList())` 으로 최종 연산을 주로 하다보니
자바16부터 `.toList`를 제공합니다.

Reply:
```java
    private static List<String> split(String text) {
        return Stream.of(text.split(CARS_NAME_SEPARATOR))
                .map(String::trim)
                .toList();
    }
```

private 메서드인데 separator 파라미터를 받을 필요가 있나요?
재사용도 안 하는 것 같은데 상수 사용하면 될 것 같아요